### PR TITLE
Minor 'How to get help' Modification

### PR DESCRIPTION
**Pull request by PrernaSingh587**
Made some changes in the How to get help section under User docs.

PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs`
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

**Comment by PrernaSingh587** [comment]
@viniciusdc Hi, could you give some feedback?

**Review by viniciusdc — COMMENTED** [review]
LGTM

File: src/user/how_to_get_help.rst [hunk 1]
```diff
@@ -1,32 +1,29 @@
-How to get help in conda-forge
+How to reach out to us
```
[the remark this answers is not in the document]
Comment (PrernaSingh587) [reply src/user/how_to_get_help.rst:1]: OKay, I will correct this one

**Comment by PrernaSingh587** [comment]
> LGTM

Thanks! I have corrected what you suggested

**Review by viniciusdc — APPROVED** [review]
Great, just need to wait for the CI tests to pass now

**Pull request merged**